ContentTransformer/ContentTransformInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace EMS\CoreBundle\ContentTransformer;
+
+interface ContentTransformInterface
+{
+    public function canTransform(ContentTransformContext $contentTransformContext): bool;
+    public function transform(string $input): string;
```

Comment:
To be discussed, but I think every function should have the ContentTransformContext as parameter